src/fe-text/terminfo-core.c

```diff
@@ -535,6 +537,11 @@ void terminfo_cont(TERM_REC *term)
 	if (term->TI_smkx)
 		tput(tparm(term->TI_smkx));
 
+	if (bracketed_paste_enabled) {
+		bracketed_paste_enabled = FALSE;
+		term_set_bracketed_paste_mode(TRUE);
```

Comment:
I guess that's possible. I kinda dislike sending the same sequence over and over though... currently any random setting change will re-set that mode in the terminal